src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {

```

Comment:
The class `Rectangle` should not begin with an empty line after the class declaration.